pages.pt_BR/linux/apt-get.md

```diff
@@ -23,6 +23,10 @@
 
 `apt-get upgrade`
 
+-  Limpar o repositório local — removendo os arquivos de pacotes (.deb) de downloads interrompidos que não podem mais ser baixados.
```

Comment:
Sorry, but the lint error for the english page makes this visible to me. The example descriptions should end with `:`, like the english page:
```suggestion
-  Limpar o repositório local — removendo os arquivos de pacotes (.deb) de downloads interrompidos que não podem mais ser baixados:
```

Reply:
Yes @navarroaxel , it was also visible to me. I'm going to fix this now.
Thank you.